addons/Text_Coloring/src/presenter.js

```diff
@@ -813,6 +825,7 @@ function AddonText_Coloring_create() {
 
     presenter.validateModel = function (model) {
         var validatedColors = presenter.validateColors(model.colors);
+        var validatedIsNotActivity = ModelValidationUtils.validateBoolean(model['isNotActivity']);
```

Comment:
Lepiej aby metoda ta nie rozdzielała spójnej całości czyli walidacji `validatedColors` a potem sprawdzenia wyniku tej walidacji.
Jeżeli ticket wróci to proszę o poprawkę.